node/src/components/chainspec_loader.rs

```diff
@@ -81,19 +63,9 @@ pub(crate) enum Event {
 impl Display for Event {
     fn fmt(&self, formatter: &mut Formatter<'_>) -> fmt::Result {
         match self {
-            Event::Initialize {
-                maybe_highest_block,
-            } => {
-                write!(
-                    formatter,
-                    "initialize(maybe_highest_block: {})",
-                    maybe_highest_block
-                        .as_ref()
-                        .map_or_else(|| "None".to_string(), |block| block.to_string())
-                )
+            Event::Initialize => {
+                write!(formatter, "initialize",)
```

Comment:
```suggestion
                write!(formatter, "initialize")
```

(Or even without `{}`, like the next case.)